clang/test/Driver/darwin-embedded-search-paths.c

```diff
@@ -0,0 +1,55 @@
+// UNSUPPORTED: system-windows
+//   Windows is unsupported because we use the Unix path separator `/` in the test.
+
```

Comment:
As written, this requires an `aarch64-registered-target`. I think you can avoid that though by turning these into `-###` style tests, and checking the include paths passed from the driver to cc1, instead of checking the user-facing diagnostics that `-v` dumps to stderr.

Reply:
Also copied from clang/test/Driver/darwin-subframeworks.c, but let me look at changing these.